src/main/java/frc/robot/Alignable.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+import java.util.function.BiFunction;
+
+public enum Alignable {
```

Comment:
Lets separate the two concerns: One for the indication by the driver (where to aim) and the other for the drive calculations (angle and speed).
Have the enum be the location, and create a class (say, AutoAlignment) that does the calculation.

Reply:
done